dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -124,12 +121,24 @@ public void runWESCommand(String jsonString, File localPrimaryDescriptorFile, Fi
         // Including it twice should not cause a problem
         workflowAttachment.add(localPrimaryDescriptorFile);
 
-        addFilesToWorkflowAttachment(workflowAttachment, this.zippedEntry, tempDir);
 
-        File jsonInputFile = new File(jsonString);
+        addFilesToWorkflowAttachment(workflowAttachment, zippedEntry, tempDir);
+
         // add the input file so the endpoint has it; not sure if this is needed
+        File jsonInputFile = new File(jsonInputFilePath);
         workflowAttachment.add(jsonInputFile);
 
+        String jsonString = null;
+        try {
+            jsonString = abstractEntryClient.fileToJSON(jsonInputFilePath);
+        } catch (IOException ex) {
+            System.out.println("Could not construct JSON from " + jsonInputFilePath + ". Request will not be sent to WES endpoint. "
```

Comment:
I don't think you need this message; exceptionMessage will display it via LOG.

Reply:
done